core/trino-main/src/test/java/io/trino/sql/query/TestColumnMask.java

```diff
@@ -196,25 +196,6 @@ public void testConditionalMask()
                 .matches("VALUES (NULL), CAST('-781' AS BIGINT)");
     }
 
-    @Test
-    public void testMultipleMasksOnSameColumn()
```

Comment:
I would keep the test and validate that in case of multiple masks it fails as expected

Reply:
It's not possible, since TestingAccessControl allows a single mask per column.